### PR TITLE
Add mtu and accessConfig flags to subnet create commands

### PR DESCRIPTION
**Pull request by ismirlia**
This pull request makes the necessary updates to the sdk to support these two new options in subnet create.

This is currently a draft pull request and API changes are not in place, so this won't compile.

### CODE REVIEW

File: helpers/constants.go [hunk 1]
```diff
@@ -243,10 +234,12 @@ const (
...
 
-	//Timeout values for Power VS -
-
+	// Timeout values for Power VS -
 	PICreateTimeOut = 5 * time.Minute
 	PIUpdateTimeOut = 5 * time.Minute
 	PIDeleteTimeOut = 3 * time.Minute
 	PIGetTimeOut    = 2 * time.Minute
+
+	// Stratos CRN
```
Comment (michaelkad) [inline helpers/constants.go:243]:
```suggestion
	// Stratos region prefix
```

File: helpers/constants.go [hunk 2]
```diff
@@ -243,10 +234,12 @@ const (
...
+	PIStratos = "satloc"
```
Comment (michaelkad) [inline helpers/constants.go:244]:
```suggestion
	PIStratosRegionPrefix = "satloc"
```

File: clients/instance/ibm-pi-network.go [hunk 3]
```diff
@@ -56,6 +57,17 @@ func (f *IBMPINetworkClient) GetAll() (*models.Networks, error) {
 
 // Create a Network
 func (f *IBMPINetworkClient) Create(body *models.NetworkCreate) (*models.Network, error) {
+	if strings.Contains(f.session.Options.Region, helpers.PIStratos) {
```
[the remark this answers is not in the document]
Comment (ismirlia) [reply clients/instance/ibm-pi-network.go:60]: Sure I'll make that change.

File: helpers/constants.go (hunk 2 above)
Comment (ismirlia) [reply helpers/constants.go]: made changes

File: helpers/constants.go (hunk 1 above)
Comment (ismirlia) [reply helpers/constants.go]: made changes

**Review by michaelkad — APPROVED** [review]

**Review by powervs-ibm — APPROVED** [review]
Looks good

**Pull request merged**